docs/basics/beginners-contributing-guide.md

```diff
@@ -0,0 +1,119 @@
+
+0. Verify the new [upstream](https://github.com/selfhostedshow/wiki) repository you've specified for your fork, type `git remote -v`, the output should look like the below.
+
+```bash
+origin  https://github.com/YOURGITHUBSUSERNAME/wiki (fetch)
+origin  https://github.com/YOURGITHUBSUSERNAME/wiki (push)
+upstream        https://github.com/selfhostedshow/wiki (fetch)
+upstream        https://github.com/selfhostedshow/wiki (push)
+```
+
+#### 1.2 Start the Wiki Docker Container
+This section will start the wiki docker container in the local environment.
+
+__Note__: The below path may be different, depending on where the wiki was cloned to.
+
+__Note__: -f will define a docker-compose file and -d ("detached" mode) will start the wiki it in the background
+
+0. Type `sudo docker-compose -f ~/PATHTOWIKI/wiki/docker-compose.yml up -d` this will start the wiki in docker.
+
+0. Type `docker ps` to verify that the wiki container is running.
+
+0. Navigate to `http://DockerIP:8000` the "dev" branch wiki website will then open.
+
+0. Now edit the wiki site in your editor of choice and when you **save** the file, the site will be updated with the changes that are made enabling real time (more or less) verifying what the site will look like with the changes.
+
+#### 1.3 Merging an local repository into your Fork
+This Section will "merge" the local changes to your fork (your github repo that was made above in [section 1.1](https://github.com/selfhostedshow/wiki/tree/dev/docs\basics/beginners-contributing-guide/#11-fork-clone-the-self-hosted-show-wiki))
+
+0. Type `git add YOURFILENAME1.md YOURFILENAME2.md` to add the changed files to your commit.
+
+0. Type `git commit -m 'updated x,y,z'` to commit the changes to your fork with comments.
+
+0. Type  `git push origin dev` to push your commit to your fork.
+
+0. Type your username for github.
+
+0. Type your password for github.
+
+#### 1.3 Make a New Pull Request on the GitHub Website
+This section will create a new pull request from your Github repo to the [Upstream](https://github.com/selfhostedshow/wiki) wiki site.
+
+0. Navigate to the Github [wiki](https://github.com/selfhostedshow/wiki) web interface.
+
+0. Click ![newpulrequest](images/newpullrequrest.PNG) in the top(ish) left, verify the the branch is set to "dev". 
+
+0. Under Compare changes click "compare across forks" link, verify the **base** repository is the [upstream](https://github.com/selfhostedshow/wiki) site and the head repository is your repo.
+
+0. The changes that were made should come automatically, make the comment succinct and meaningful.
+
+0. Submit a pull request by clicking ![createpullrequest](images/createpullrequest.PNG).
+
+0. The commit has now been added to the pull request the Wiki-Admins will review and if approved will get pushed to the master branch for publishing.
+
```

Comment:
Something i'd be interested in here which would be quite useful would be what to do when there's feedback. Mostly in the fact that there's no need to recreate branches / whatever, just push to new ones (and please don't force!)

Reply:
@RealOrangeOne thanks for all the feed back. for this one ya that sounds good to me but again what do we want to do when there is feed back point them to github? also dont no what "(and please don't force!)" means again sorry new to git/github

Reply:
Force pushing basically overrides the commit history for the PR, and makes reviews more annoying. 

When there's feedback, people just commit and push to the same branch they originally submitted, but with changes. And the commit message shouldn't be something generic like "changes from review", because that's not very descriptive. Submitting PRs / updates with multiple commits is completely fine!

Reply:
ok i can add that info as a note to that page

Reply:
updated and moved to wiki folder